staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -296,9 +299,6 @@ func WriteWatchCacheSizes(watchCacheSizes map[schema.GroupResource]int) ([]strin
 	var cacheSizes []string
 
 	for resource, size := range watchCacheSizes {
-		if size < 0 {
```

Comment:
disallow < -1?

Reply:
I'm not sure if this changes something, as any negative value will be treated as undecorated storage

Reply:
I think we should stick to -1 and avoid people using -12347 as an example.